frontend/src/features/dataset/DatasetHeader.tsx

```diff
@@ -13,11 +13,13 @@ import Button from '../../chrome/Button'
 
 export interface DatasetHeaderProps {
   qriRef: QriRef
+  noBorder?: boolean
```

Comment:
avoid defining props through negation whenever possible. refactor this to `border` & add the `border` prop to all call sites that currently use a border.